rs/node_rewards/canister/src/main.rs

```diff
@@ -4,28 +4,27 @@ use ic_cdk::{init, post_upgrade, pre_upgrade, spawn, update};
 use ic_nervous_system_canisters::registry::RegistryCanister;
 use ic_nns_constants::GOVERNANCE_CANISTER_ID;
 use ic_node_rewards_canister::canister::NodeRewardsCanister;
+use ic_node_rewards_canister::registry::RegistryClient;
```

Comment:
So why switch the implementation, instead of enhancing the one already in the monorepo?